pkg/connection/manager_test.go

```diff
@@ -87,7 +84,8 @@ func TestManagerConnectStore(t *testing.T) {
 				},
 			},
 			want: want{
-				err: errors.Wrapf(kerrors.NewNotFound(schema.GroupResource{}, fakeConfig), errGetStoreConfig),
+				// TODO(negz): Can we test that this is kerrors.NewNotFound?
```

Comment:
I tried implementing a custom EquateErrors based on `errors.As`, but unfortunately I couldn't make it work, so I guess we're stuck with `cmptopts.AnyError`.